iterators.cpp

```diff
@@ -0,0 +1,66 @@
+#include <iostream>
+#include <vector>
+#include <iterator>
+
+
+//reverse iteration
+
+int reverse_order() {
+
+    std::vector<int> v = {1, 2, 3, 4};
+    auto it = v.rbegin();
+    auto start = v.rend();
+
+    for(; it != start; it++) {
+        std::cout << *it << std::endl;
+    }
+
+    return 0;
```

Comment:
If you don't want to return anything you can make the function

    void reverse_order() {
      ..
    }

and remove the `return 0`.